src/commands/community/nft/ticker.ts

```diff
@@ -35,18 +43,234 @@ import { CommandError } from "errors"
+  const { messageOptions } = await composeCollectionTickerEmbed({
+    msg,
+    authorId,
+    collectionAddress,
+    chain,
+    ...(days && { days: +days }),
+  })
+  await msg.edit(messageOptions)
+}
+
+async function viewTickerInfo(
+  msg: Message,
+  authorId: string,
+  params: { collectionAddress: string; chain: string }
+) {
+  const { collectionAddress, chain } = params
+  const { messageOptions } = await composeCollectionInfoEmbed(
+    msg,
+    authorId,
+    collectionAddress,
+    chain
+  )
+  await msg.edit(messageOptions)
+  await msg.removeAttachments()
+}
+
+function composeTickerSelectionResponse(
+  suggestions: ResponseCollectionSuggestions[],
+  symbol: string,
+  msg: Message
+) {
+  const opt = (s: ResponseCollectionSuggestions): MessageSelectOptionData => ({
+    label: `${s.name} (${s.symbol})`,
+    value: `${s.address}_${s.chain}_${msg.author.id}`,
+  })
+  const selectRow = composeDiscordSelectionRow({
+    customId: "nft_tickers_selection",
+    placeholder: "Make a selection",
+    options: suggestions.map(opt),
+  })
+
+  const found = suggestions.map((s) => `**${s.name}** (${s.symbol})`).join(", ")
+  return {
+    messageOptions: {
+      embeds: [
+        composeEmbedMessage(msg, {
+          title: `${defaultEmojis.MAG} Multiple collections found`,
+          description: `Multiple collections found for \`${symbol}\`: ${found}.\nPlease select one of the following collection`,
+        }),
+      ],
+      components: [selectRow, composeDiscordExitButton(msg.author.id)],
+    },
+    commandChoiceOptions: {
+      userId: msg.author.id,
+      guildId: msg.guildId,
+      channelId: msg.channelId,
+      handler: tickerSelectionHandler,
+    },
+  }
+}
+
+async function composeCollectionInfoEmbed(
+  msg: Message,
+  authorId: string,
+  collectionAddress: string,
+  chain: string
+) {
+  const { data, ok, curl, log } = await community.getNFTCollectionMetadata(
+    collectionAddress,
+    chain
+  )
+  if (!ok) {
+    throw new APIError({ message: msg, curl: curl, description: log })
+  }
+  if (!data) {
+    throw new CommandError({
+      message: msg,
+      description: "The collection does not exist. Please choose another one.",
+    })
+  }
+  const blank = getEmoji("blank")
+  const symbol = `${data.symbol?.toUpperCase() ?? "-"}`
+  const address = data.address ? `${shortenHashOrAddress(data.address)}` : "-"
+  const name = `${data.name ?? "-"}`
+  const desc = `${data.description ?? "-"}`
+  const discord = `${data.discord ?? "-"}`
+  const twitter = `${data.twitter ?? "-"}`
+  const website = `${data.website ?? "-"}`
+  const ercFormat = `${data.erc_format ?? "-"}`
+  const marketplaces = data.marketplaces
+    ? data.marketplaces.map((m: string) => getEmoji(m)).join(" ")
+    : "-"
+  let createdTime = "-"
+  if (data.created_at) {
+    createdTime = dayjs(data.created_at).format("DD/MM/YYYY")
+  }
+  const fields = [
+    {
+      name: "Symbol",
+      value: `${symbol}`,
```

Comment:
```suggestion
      value: symbol,
```
chi cho cực dị 😄 , mấy cái dưới nữa

Reply:
tui sửa r nha 😅